src/actions/snackbar.js

```diff
@@ -0,0 +1,11 @@
+export const SUMMON_SNACKBAR = 'SUMMON_SNACKBAR';
+export const DISMISS_SNACKBAR = 'DISMISS_SNACKBAR';
+
+export const summonSnackbar = message => ({
```

Comment:
I'm wondering whether this should be called snackbar. Snackbar is the "just" the visualization concept, not the underlying idea of showing messages to users. Maybe that's just an OCD thing for me, but maybe using "messages"/"message" in the redux level seems more usable when a different pattern appears.

Reply:
That makes a lot of sense to me – the current terminology kind of shoehorns this into a single-use Redux action/reducer instead of being a generalized interface for issuing messages to the user regardless of the component the message finally renders at. 

With that in mind, I got to grepping around and it looks like we already have setMessage/clearMessage Redux actions that can definitely be utilized to that end with some modifications.

I'm going to throw up another commit here in a sec that refines some of the logic behind the setMessage/clearMessage actions so that they can be more broadly applied.

Reply:
Reviewing it again this PR is so much nicer!